communication/src/main/java/PNC.java

```diff
@@ -0,0 +1,31 @@
+import org.jboss.da.communcation.pnc.ReadResource;
+import org.jboss.da.communcation.pnc.authentication.PNCAuthFilter;
+import org.jboss.da.communcation.pnc.entity.Product;
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.client.jaxrs.ResteasyWebTarget;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Enumeration;
+import java.util.Properties;
+import java.util.ResourceBundle;
+
+public class PNC {
+    /**
+     * Static factory to create an instance of PNCInterface
+     *
+     * @return instance of PNCInterface
+     */
+    public static PNCInterface getPNCInstance(boolean isAuthenticated) {
```

Comment:
I'm still thinking in non-containerized Java land :) Thanks for all the input! Will keep that in mind for my next PR!